odl_video/settings.py

```diff
@@ -289,8 +289,12 @@ def get_all_config_keys():
 EMAIL_SUPPORT = get_string('ODL_VIDEO_SUPPORT_EMAIL', 'support@example.com')
 DEFAULT_FROM_EMAIL = get_string('ODL_VIDEO_FROM_EMAIL', 'webmaster@localhost')
 
-MAILGUN_URL = get_string('MAILGUN_URL', 'https://api.mailgun.net/v3/video.odl.mit.edu')
+MAILGUN_URL = get_string('MAILGUN_URL', None)
+if not MAILGUN_URL:
```

Comment:
I think these checks might not be necessary because `MAILGUN_URL` and `MAILGUN_KEY` are included in the `MANDATORY_SETTINGS` at the bottom of settings, and are checked in [apps.py](https://github.com/mitodl/odl-video-service/blob/4ad7018b930221c67b1b8e69e5a0d7986f2cef16/ui/apps.py#L18)

Reply:
Ah, you're right

Reply:
Actually we should still remove the default MAILGUN_URL here and move it into .env, but I'll open another PR for that